style.scss

```diff
@@ -414,7 +414,7 @@ body >footer {
 					color: $purple-highlight;
 			}
 				
-			&.twitter::before { content: "\e908"; }
+			&.twitter::before { content: "\e909"; }
```

Comment:
Yeah, I know. I had to regenerate the font though and I don't know how to regenerate it without changing the numbers. Any ideas?